src/main/webapp/config.js

```diff
@@ -0,0 +1,3 @@
+var config = {
+  API_KEY : "AIzaSyB-kjcAxxbrQeJJEvSg5XgkTmGI4aZLS5E"
```

Comment:
Is this hidden through our .gitignore file?

Reply:
Yeah I deleted this config.js in a previous PR. This is the old API key but delete this file anyways Erick!